cmd/main.go

```diff
@@ -179,6 +179,21 @@ func parseConfig() (Config, error) {
 		retval.reconcile = *retval.ctrlConfig.Reconcile
 	}
 
+	if retval.ctrlConfig.EntryIDPrefix != "" && retval.ctrlConfig.EntryIDPrefixCleanup != nil && retval.ctrlConfig.EntryIDPrefix == *retval.ctrlConfig.EntryIDPrefixCleanup {
```

Comment:
I think you want to do this equality comparison after you've normalized these values with the dot suffix.

Reply:
Normalized and moved to reduce comparisons